etna/core/models/mixins.py

```diff
@@ -160,31 +159,14 @@ class Meta:
     ]
 
     api_fields = [
-        # APIField("hero_image"),
         APIField("hero_image_caption", serializer=RichTextSerializer()),
         APIField(
-            "hero_image_jpg",
-            serializer=ImageRenditionField(
-                "fill-1200x480|format-jpeg|jpegquality-60", source="hero_image"
-            ),
+            "hero_image",
+            serializer=DetailedImageSerializer("fill-1200x480"),

```

Comment:
```suggestion
            serializer=ImageSerializer("fill-1200x480"),

```

I don't think we need all the fields for a hero image.